Framework/Kernel/test/PropertyManagerTest.h

```diff
@@ -481,6 +481,15 @@ class PropertyManagerTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(i, 655);
   }
 
+  void test_char_array() {
```

Comment:
Is there a missing `TS_ASSERT` here - It looks like the test doesn't actually do anything? Is this supposed to test whether leaving it blank raises an exception?